website/templates/coach_dashboard.html

```diff
@@ -507,7 +507,20 @@ <h4 class="card-title"> Athletes</h4>
                         <tbody>
                           <tr>
                             
-                            <td>
```

Comment:
Where is athlete_data coming from? and all_users?

Reply:
placeholders for corresponding data (eg sleep, nutrition etc.) I don't know what to call them exactly hahah

Reply:
and I wanted to get a list of all athletes in a team. not quite sure how to do it though

Reply:
Oh ok cool. I can show you how to do it, it's actually quite simple